classes/class-wc-connect-nux.php

```diff
@@ -60,6 +64,53 @@ public function check_notice_dismissal() {
 			$notices[ $notice ] = true;
 			update_option( 'wc_connect_nux_notices', $notices );
 		}
-	}
 
-}
\ No newline at end of file
+		private function init_pointers() {
+			add_filter( 'wc_services_pointer_woocommerce_page_wc-settings', array( $this, 'register_add_service_to_zone_pointer' ) );
+		}
+
+		public function show_pointers( $hook ) {
+			$pointers = apply_filters( 'wc_services_pointer_' . $hook, array() );
```

Comment:
Let's document our hooks (note: I'm not too sure we've been great about this so far) using the WP standards: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters